extensions/wikia/Search/templates/WikiaSearch_CrossWiki_result.php

```diff
@@ -14,9 +11,14 @@
 		<p class="description"><?= $description; ?></p>
 
 		<ul class="wiki-statistics subtle">
-			<li><?= $pagesMsg ?></li>
-			<li><?= $imgMsg ?></li>
-			<li><?= $videoMsg ?></li>
+			<?
+				$shortenedPagesCount = $wg->Lang->shortenNumberDecorator( $pagesCount);
+				$shortenedImagesCount = $wg->Lang->shortenNumberDecorator( $imagesCount);
+				$shortenedVideosCount = $wg->Lang->shortenNumberDecorator( $videosCount);
+			?>
+			<li><?= wfMessage( 'wikiasearch2-pages' )->params( $shortenedPagesCount->decorated, $shortenedPagesCount->rounded )->escaped(); ?></li>
+			<li><?= wfMessage( 'wikiasearch2-images' )->params( $shortenedImagesCount->decorated, $shortenedImagesCount->decorated )->escaped(); ?></li>
+			<li><?= wfMessage( 'wikiasearch2-videos' )->params( $shortenedVideosCount->decorated, $shortenedVideosCount->decorated )->escaped(); ?></li>
```

Comment:
Why do we pass `rounded` as the second argument for pages but `decorated` for images and videos?